src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -56,10 +38,13 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
   import system.dispatcher
   val log = Logging(system, getClass)
 
+  // GAWB-1286
```

Comment:
All I can really say with confidence is that ProfileClient doesn't look enough like OauthService (for example) to receive app.thurloeDao and whatnot. Suggestions welcome.